.github/workflows/update-deps.yml

```diff
@@ -53,6 +66,7 @@ jobs:
           fi
 
           if ${changed}; then
+            cabal update
```

Comment:
IDK why this is necessary... but without it, the `cabal freeze` call would not find any hackage-supplied packages (only the `co-log` that we request explicitly in the cabal.project would be available). :man_shrugging: